src/tests/test_magic_cmd.py

```diff
@@ -14,6 +14,20 @@
 )
 
 
+def _get_row_string(row, column_name):
+    """
+    Helper function to retrieve the string value of a specific column in a table row.
+
+    Args:
```

Comment:
`Args` -> `Parameters`. You can look at some existing functions to check the convention